### ISSUE TITLE
Decommission library-staging[12].lib.princeton.edu

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/princeton_ansible#5406]
### What maintenance needs to be done?

Decommission library-staging[12].lib.princeton.edu

### Why is this maintenance needed?

These vms are no longer needed, since we no longer host and maintain the drupal library website.

### Acceptance criteria
 - [x] These VMs are not referenced in load balancer configurations
 - [x] These VMs are not in the inventory
 - [x] delete library-staging1.lib.princeton.edu vm
 - [x] delete library-staging2.lib.princeton.edu vm
 - [x] delete DNS for the VMs
 - [x] delete firewall rules

**Comment by dphillips-39** [comment]
Both VM's were disconnected from the network and will be powered off on Oct 10, 2024

**Comment by dphillips-39** [comment]
This is blocked by https://github.com/pulibrary/princeton_ansible/issues/5531

**Comment by dphillips-39** [comment]
Blocked by #2072

**Comment by VickieKarasic** [comment]
@dphillips-39 We are aiming to power these down on 12/19/24 for 30 days and then fully decommission in the new year.

**Comment by dphillips-39** [comment]
VM's not referenced in load balancer configurations file
